posthog/settings/async_migrations.py

```diff
@@ -1,9 +1,11 @@
-from posthog.settings.base_variables import E2E_TESTING, TEST
+from posthog.settings.base_variables import DEBUG, E2E_TESTING, TEST
 from posthog.settings.overrides import cmd
 from posthog.settings.service_requirements import SKIP_SERVICE_VERSION_REQUIREMENTS
 from posthog.settings.utils import get_from_env, str_to_bool
 
-_default_skip_async_migrations_setup = TEST or E2E_TESTING or SKIP_SERVICE_VERSION_REQUIREMENTS or cmd != "runserver"
+_default_skip_async_migrations_setup = (
+    False if DEBUG else (TEST or E2E_TESTING or SKIP_SERVICE_VERSION_REQUIREMENTS or cmd != "runserver")
```

Comment:
Is this a reasonable change @yakkomajuri 

I was surprised I needed to skip skipping when running in DEBUG...

Reply:
Hmmm. looks like it breaks asyncmigrations CI so I guess not a reasonable change 🤣

Reply:
hmm yeah so this is here because otherwise it will scream at people on dev for not having completed an async migration or something of the sort. Best is just to set `SKIP_ASYNC_MIGRATIONS_SETUP` locally